team-telnyx/webrtc
# feat(WEBRTC-530): add unified-plan and add transceivers

- add unified-plan and add transceivers
 - update some vulnerability dependencies

## 📝 To Do


## ✋ Manual testing

1. Run `npm i` on the root folder
2. Navigate to `package/js` and run `npm i` and `npm run build` and `npm run test` it should returns green.
3. Navigate to `package/js/examples/vanilla/index.html` change the line 18 to be like this 
```
 
    <script type="text/javascript" src="../../lib/bundle.js"></script>
```
4. Disable production checkbox
5. open index.html on Safari, Chrome, Firefox, Edge and try to make audio and video call it should show the Local and Remote video for you without any warning or error on console.log.

## 🦊 Browser testing

### Desktop


## 📸 Screenshots

| Description | Screenshot |
| ----------- | ---------- |
| Desktop     |![image](https://user-images.githubusercontent.com/16343871/120359866-8a139700-c2de-11eb-9e3f-98afecd2106e.png)|
